### PR TITLE
Fingerprint add for the Zemismart Thread Smart Switch Binary  MT606-1

### PR DESCRIPTION
**Pull request by lelandblue**
WWST Cert submission from the Edge Builder Tool to add a fingerprint for the Zemismart Thread Smart Switch Binary  MT606-1 device

### CODE REVIEW

File: drivers/SmartThings/matter-switch/fingerprints.yml [hunk 1]
```diff
@@ -71,6 +71,12 @@ matterManufacturer:
     vendorId: 0x1312
     productId: 0x01
     deviceProfileName: light-level-colorTemperature-2710k-6500k
+#Zemismart
+  - id: "5020/43522"
+    deviceLabel: "Zemismart Thread Smart Switch"
+    vendorId: 0x139C
+    productId: 0xAA02
+   deviceProfileName: "switch-binary"
```
Comment (greens) [inline drivers/SmartThings/matter-switch/fingerprints.yml:79]:
```suggestion
    deviceProfileName: "switch-binary"
```

File: drivers/SmartThings/matter-switch/fingerprints.yml [hunk 2]
```diff
@@ -71,6 +71,12 @@ matterManufacturer:
     vendorId: 0x1312
     productId: 0x01
     deviceProfileName: light-level-colorTemperature-2710k-6500k
+#Zemismart
```
Comment (hubTab) [inline drivers/SmartThings/matter-switch/fingerprints.yml:74]:
```suggestion
# Zemismart
```

**Review by hubTab — APPROVED** [review]
Left in a suggestion, otherwise is good to go.

**Comment by lelandblue** [comment]
Closing, did not complete WWST.

**Pull request closed without merging**